status-variables.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Server Status Variables
+summary: Use status variables to see the system and session status
+---
+
+# Server Status Variables
+
+Most status variables are implemented to be compatible with MySQL. These variables give information about the global status of the server and the status of the current session.
+
+The [SHOW GLOBAL STATUS](/sql-statements/sql-statement-show-status.md) command can be used to retrieve the global status and the [SHOW SESSION STATUS](/sql-statements/sql-statement-show-status.md) command can be used to see the status of the current session.
+
+The [FLUSH STATUS](/sql-statements/sql-statement-flush-status.md) command is also support to aid MySQL compatibility.
+
+## Variable Reference
```

Comment:
```suggestion
## Variable reference
```

Reply:
`system-variables.md` is also using `Variable Reference`. Are they both wrong?

Reply:
Yes, they are both wrong here. According to our style guide, we use sentence case for all headings except for H1.

Reply:
Fixed the other one in 37a8e95c3938e2068db927c955fb1a43600bb0ad